mappings/net/minecraft/world/gen/feature/Feature.mapping

```diff
@@ -0,0 +1,91 @@
+CLASS btl net/minecraft/world/gen/feature/Feature
+	FIELD A MEGA_JUNGLE_TREE Lbtl;
+	FIELD B MEGA_PINE_TREE Lbtl;
+	FIELD C MEGA_SPRUCE_TREE Lbtl;
+	FIELD D DEFAULT_FLOWER Lbto;
+	FIELD E FOREST_FLOWER Lbto;
+	FIELD F PLAIN_FLOWER Lbto;
+	FIELD G SWAMP_FLOWER Lbto;
+	FIELD H GENERAL_FOREST_FLOWER Lbto;
+	FIELD I JUNGLE_GRASS Lbtl;
+	FIELD J TAIGA_GRASS Lbtl;
+	FIELD K GRASS Lbtl;
+	FIELD L VOID_START_PLATFORM Lbtl;
+	FIELD M CACTUS Lbtl;
+	FIELD N DEAD_BUSH Lbtl;
+	FIELD O DESERT_WELL Lbtl;
+	FIELD P FOSSIL Lbtl;
+	FIELD Q HELL_FIRE Lbtl;
+	FIELD R HUGE_RED_MUSHROOM Lbtl;
+	FIELD S HUGE_BROWN_MUSHROOM Lbtl;
+	FIELD T ICE_SPIKE Lbtl;
+	FIELD U GLOWSTONE_BLOB Lbtl;
+	FIELD V MELON Lbtl;
+	FIELD W PUMPKIN Lbtl;
+	FIELD X REED Lbtl;
+	FIELD Y FREEZE_TOP_LAYER Lbtl;
+	FIELD Z VINES Lbtl;
+	FIELD aA CORAL_CLAW Lbtl;
+	FIELD aB SEA_PICKLE Lbtl;
+	FIELD aC SIMPLE_BLOCK Lbtl;
+	FIELD aD BAMBOO Lbtl;
+	FIELD aE DECORATED Lbtl;
+	FIELD aF DECORATED_FLOWER Lbtl;
+	FIELD aG BONUS_CHEST Lbsd;
+	FIELD aH STRUCTURES Lcom/google/common/collect/BiMap;
+	FIELD aa WATERLILY Lbtl;
+	FIELD ab MONSTER_ROOM Lbtl;
+	FIELD ac BLUE_ICE Lbtl;
+	FIELD ad ICEBERG Lbtl;
+	FIELD ae FOREST_ROCK Lbtl;
+	FIELD af BUSH Lbtl;
+	FIELD ag DISK Lbtl;
+	FIELD ah DOUBLE_PLANT Lbtl;
+	FIELD ai NETHER_SPRING Lbtl;
+	FIELD aj ICE_PATCH Lbtl;
+	FIELD ak LAKE Lbtl;
+	FIELD al ORE Lbtl;
+	FIELD am RANDOM_RANDOM_SELECTOR Lbtl;
```

Comment:
RANDOM_RANDOM_SELECTOR is that a mistake?

Reply:
Nope. It selects a random selector from a list randomly